apps/services/handlers.py

```diff
@@ -270,5 +283,10 @@ async def __generate_user_document_name(cls, service_id: int, session: AsyncSess
         )
         return f"Заява на {service.service_name.lower()}"
 
+    @classmethod
```

Comment:
i added this method to separate some logic inside "download_user_document" method. Please don't forget to pull changes

Reply:
ok